pkg/docker/entry.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+
+[ -z "$NO_EDID" ] && {
+	[ -n "$EDID_HEX" ] && echo "$EDID_HEX" > /edid.hex
+	while true; do
+		v4l2-ctl --device=/dev/video0 --set-edid=file=/edid.hex --fix-edid-checksums --info-edid && break
+		echo 'Failed to set EDID. Reetrying...'
+		sleep 1
+	done
+}
+
+./ustreamer --host=0.0.0.0 $@
```

Comment:
Should we add `--persistent --dv-timings` automatically if we use edid? What do you think?

Reply:
It is started by entrypoint. Adding here will make it difficult to change if someone needs to.
Defaults values are passed from https://github.com/pikvm/ustreamer/blob/17f54a79773334b3632cd2bd5c0d1e5ac408916a/pkg/docker/Dockerfile.alpine#L28
it would be better to add it there.

Reply:
Okay